README.md

```diff
@@ -95,6 +102,7 @@
 - [Hacktoberfest Checker](https://hacktoberfestchecker.jenko.me/)
 - [Hacktoberfest Tracker Mobile App](https://play.google.com/store/apps/details?id=com.hacktoberfesttrackerapp) [github](https://github.com/KeyboardNinjas/hacktoberfest-mobileapp)
 - [HacktoberfestSwagList - List of companies giving out swag for participating in Hacktoberfest](https://hacktoberfestswaglist.com/)
+- [First Contributions Guide](https://github.com/firstcontributions/first-contributions)
```

Comment:
Are you sure that we can consider First Contributions as intermediate/advanced instead of beginners?